lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -27,12 +29,20 @@
                     this.teamsUrl = options.teamParams.teamsUrl;
                     this.languages = options.teamParams.languages;
                     this.countries = options.teamParams.countries;
-                    this.primaryButtonTitle = options.primaryButtonTitle || 'Submit';
-
-                    _.bindAll(this, 'goBackToTopic', 'createTeam');
-
-                    this.teamModel = new TeamModel({});
-                    this.teamModel.url = this.teamsUrl;
+                    this.teamsDetailUrl = options.teamParams.teamsDetailUrl;
+                    this.action = options.action;
+
+                    _.bindAll(this, 'cancelAndGoBack', 'createOrUpdateTeam');
+
+                    if (this.action === 'create') {
+                        this.teamModel = new TeamModel({});
+                        this.teamModel.url = this.teamsUrl;
+                        this.primaryButtonTitle = 'Create';
```

Comment:
Even if this is existing, it needs to be fixed-- it shouldn't have passed previous code reviews. It will only translate if some place in our codebase someone happens to have scraped this same text.